Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[workspace]
+members = ["crates/nats-stream"]
+resolver = "2"
+
+[workspace.package]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+keywords = ["data-stream", "blockchain", "cryptocurrencies"]
+edition = "2021"
+homepage = "https://fuel.network/"
+license = "Apache-2.0"
+repository = "https://github.com/FuelLabs/data-systems"
+version = "0.0.1"
+
+[workspace.dependencies]
+anyhow = "1.0.86"
+clap = { version = "4.5.4", features = ["derive"] }
+futures = "0.3.30"
+tokio = { version = "1.38.0", features = ["full"] }
```

Comment:
Is there a reason we should lock `patch` versions? My main concern is that if there is a security `patch`, this would create an extra step to catch up.

Reply:
Can you open a issue to track this comment? In this way we can merge the PR